rinex/tests/obs.rs

```diff
@@ -1173,6 +1173,41 @@ mod test {
             values
         );
     }
+    #[cfg(feature = "flate2")]
```

Comment:
my understanding is #[cfg  only applies to the following line's entire scope.
I think at the time I introduced that, it would not pick up correctly, I had to put it everywhere.

I'll double check on that

Reply:
It applies to the whole item (module, function, enum variant etc.) below it, unless you use `#!`, which makes it apply to the parent item.

Reply:
@lnicola,

I confirm that running `cargo test --default-features` does work with these suggestions, 
I'll merge this when CI gets completed